ako-gateway-api/k8s/gateway_controller.go

```diff
@@ -91,18 +91,14 @@ func (c *GatewayController) Start(stopCh <-chan struct{}) {
 		c.informers.ServiceInformer.Informer().HasSynced,
 		c.informers.NSInformer.Informer().HasSynced,
 	}
+	go c.informers.EpSlicesInformer.Informer().Run(stopCh)
+	informersList = append(informersList, c.informers.EpSlicesInformer.Informer().HasSynced)
```

Comment:
Should we be running epslice informer in NPL mode as well?

Reply:
As discussed , yes